src/Perspex.SceneGraph/Media/Geometry.cs

```diff
@@ -36,7 +36,7 @@ static Geometry()
         /// <summary>
         /// Gets the platform-specific implementation of the geometry.
         /// </summary>
-        public IGeometryImpl PlatformImpl
+        public virtual IGeometryImpl PlatformImpl
```

Comment:
@grokys I wanted to implement some king of lazy initialization of GeometryImpl before first get (or after it's state is changed). If you have a better  idea how to achieve that - we can change it.